pages/trial-locations.vue

```diff
@@ -0,0 +1,302 @@
+          </span>
+        </div>
+        <br />
+        <h2>Year: {{ sliderYear[0] }} - {{ sliderYear[1] }}</h2>
+        <div class="p-2">
+          <Slider
+            name="slider"
+            v-model="numberRangeValue"
+            :min="0"
+            :max="8"
+            :format="getYearLabel"
+            showTooltip="drag"
+            :merge="1"
+            @change="filterDates()"
+            :lazy="false"
+          />
+        </div>
+        <!-- Display slider years below the slider -->
+        <div class="slider-years">
+          <span
+            v-for="(year, index) in sliderYears"
+            :key="index"
+            class="slider-year"
+            >{{ year }}</span
+          >
+        </div>
+        <br /><br />
+      </div>
+      <div id="map-wrapper">
+        <client-only>
+          <LMap style="height: 100%; width: 100%" :zoom="zoom" :center="center">
+            <LTileLayer :url="url" :attribution="attribution"></LTileLayer>
+
+            <LMarker
+              v-for="(marker, index) in activeMarkers"
+              :key="index"
+              :lat-lng="marker.longLat"
+            >
+              <LPopup class="adapted-popup">
+                <h2>{{ marker.location }}</h2>
+                <br />
+                <div
+                  :class="
+                    marker.trials.length > 1
+                      ? 'witch-scroller'
+                      : 'no-witch-scroller'
+                  "
+                >
+                  <div v-for="(trial, index) in marker.trials" :key="index">
+                    <strong>{{ trial.witchName }}</strong
+                    ><br />
+                    Trial Date: {{ trial.date }}<br />
+                    Trial Year: {{ trial.year }}<br />
+                    <a :href="trial.link" target="_blank">More Info</a
+                    ><br /><br />
+                  </div>
+                </div>
+              </LPopup>
+              <LIcon
+                :icon-size="[25, 38]"
+                :icon-anchor="iconAnchor"
+                :iconUrl="getIcon(marker)"
+                :shadowUrl="shadowUrl"
+                :shadowSize="[32, 22]"
+                :shadowAnchor="shadowAnchor"
+              >
+              </LIcon>
+            </LMarker>
+          </LMap>
+        </client-only>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { SPARQLQueryDispatcher } from "~/assets/js/SPARQLQueryDispatcher";
+import Swal from "sweetalert2";
+import Slider from "@vueform/slider";
+
+definePageMeta({
+  layout: "default",
+});
+
+export default {
+  components: { Slider },
+  data: () => ({
+    sparqlUrl: "https://query.wikidata.org/sparql",
+    url: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+    attribution:
+      'Map data © <a href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a>. Historical Maps Layer, 1919-1947 from the <a href="http://maps.nls.uk/projects/api/">NLS Maps API</a>',
```

Comment:
I think the copyright symbol ehre needs to be url encoded. I think the code is &copy; from memory

Reply:
Brilliant, thank you